src/Asset/Service/Data/CustomMetadataService.php

```diff
@@ -35,6 +35,12 @@
 {
     use ElementProviderTrait;
 
+    private const DEFAULT_METADATA = [
```

Comment:
Can we make that public?
Then  i could use is here as well, and we do not need do Hard Code this twice? 
https://github.com/pimcore/studio-backend-bundle/blob/91c9e46213fd2baa33005312620598f634324ff0/src/Grid/Column/Collector/Asset/MetadataCollector.php#L61

Or a getter would also work!

Reply:
Certainly, even moved them to the interface.